ouroboros-network-framework/test/Test/Ouroboros/Network/Subscription.hs

```diff
@@ -32,6 +32,11 @@ import           Data.Void (Void)
 import           Data.Word
 import qualified Network.DNS as DNS
 import qualified Network.Socket as Socket
+#if !defined(mingw32_HOST_OS)
+import qualified Network.Socket.ByteString.Lazy as Socket (recv, sendAll)
+#else
+import qualified System.Win32.Async.Socket.ByteString.Lazy as Win32.Async
+#endif
```

Comment:
If we use the same qualification, we can avoid using `CPP` below.  `win32-network` modules and api where structured after the `network` package for this purpose.

```suggestion
#if !defined(mingw32_HOST_OS)
import qualified Network.Socket.ByteString.Lazy as Socket (recv, sendAll)
#else
import qualified System.Win32.Async.Socket.ByteString.Lazy as Socket (recv, sendAll)
#endif
```